include/picongpu/param/density.param

```diff
@@ -266,13 +266,13 @@ namespace densityProfiles
 
             /* triangle function example
              * for a density profile from 0 to 400 microns */
-            float_64 s = 1.0 - 5.0 * math::abs(y - 0.2);
+            float_X s = float_X( 1.0 ) - float_X( 5.0 ) * math::abs(y - 0.2);
```

Comment:
depending of which precision is needed you must cast the result of `abs` or the param of `abs`

Reply:
we really need literals, e.g. a `_X`  and `_3X` :-)
http://en.cppreference.com/w/cpp/language/user_literal